packages/synapse-react-client/src/style/base/_base.scss

```diff
@@ -32,6 +32,7 @@ html {
 body {
   color: SRC.$gray-regular;
   font-family: 'DM Sans';
+  accent-color: SRC.$secondary-action-color;
```

Comment:
Any non-MUI checkbox will fall back to the native browser input, to which we can apply some style with the relatively new [accent-color](https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color) CSS property, which looks decent!

Reply:
![image](https://github.com/user-attachments/assets/6c059426-4f29-432c-ae48-8c67ad4093bd)